java/test/src/main/java/test/Ice/faultTolerance/Server.java

```diff
@@ -17,20 +15,22 @@ public void run(String[] args) {
     properties.setProperty("Ice.Package.Test", "test.Ice.faultTolerance");
     try (com.zeroc.Ice.Communicator communicator = initialize(properties)) {
       int port = 0;
-      PrintWriter out = getWriter();
       for (String arg : remainingArgs) {
         if (arg.charAt(0) == '-') {
+          usage();
```

Comment:
This function showed up as unused code, but it seems like we just forgot to use it.
If you look in the matching `Client.java` for this test, it looks very similar to this file, but it has `usage()` calls in these locations.